config/dispatcher/dispatcher.ex

```diff
@@ -1,532 +1,566 @@
     Proxy.forward conn, path, "http://cache/contact-punten/"
   end
 
-  match "/adressen/*path" do
+  match "/adressen/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/adressen/"
   end
 
-  match "/functionaris-status-codes/*path" do
+  match "/functionaris-status-codes/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/functionaris-status-codes/"
   end
 
   #################################################################
   # personeelsdatabank
   #################################################################
-  match "/employee-datasets/*path" do
+  match "/employee-datasets/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/employee-datasets/"
   end
 
-  match "/employee-period-slices/*path" do
+  match "/employee-period-slices/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/employee-period-slices/"
   end
 
-  match "/employee-observations/*path" do
+  match "/employee-observations/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/employee-observations/"
   end
 
-  match "/employee-time-periods/*path" do
+  match "/employee-time-periods/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/employee-time-periods/"
   end
 
-  match "/educational-levels/*path" do
+  match "/educational-levels/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/educational-levels/"
   end
 
-  match "/working-time-categories/*path" do
+  match "/working-time-categories/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/working-time-categories/"
   end
 
-  match "/employee-legal-statuses/*path" do
+  match "/employee-legal-statuses/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/employee-legal-statuses/"
   end
 
-  match "/employee-unit-measures/*path" do
+  match "/employee-unit-measures/*path", %{ layer: :resources, accept: %{ json: true } } do
     Proxy.forward conn, path, "http://cache/employee-unit-measures/"
   end
 
   #################################################################
   # adressenregister
   #################################################################
-  match "/adressenregister/*path" do
+  match "/adressenregister/*path", %{ layer: :api_services, accept: %{ any: true } } do
     Proxy.forward conn, path, "http://adressenregister/"
   end
 
   #################################################################
   # Reports
   #################################################################
-  match "/reports/*path" do
+  match "/reports/*path", %{ layer: :resources, accept: %{ any: true } } do
```

Comment:
Shouldn't this be accept json?

Reply:
Could be, I was not really sure as I thought that reports also accepted files